hedera-mirror-importer/src/main/resources/db/migration/v2/V2.0.0__time_scale_init.sql

```diff
@@ -228,13 +252,16 @@ create table if not exists token
     initial_supply         bigint                 not null,
     kyc_key                bytea,
     kyc_key_ed25519_hex    varchar                null,
+    max_supply             bigint                 not null default 9223372036854775807, -- max long
     modified_timestamp     bigint                 not null,
     name                   character varying(100) not null,
     supply_key             bytea,
     supply_key_ed25519_hex varchar                null,
+    supply_type            character varying(10)  not null default 'INFINITE',
     symbol                 character varying(100) not null,
     total_supply           bigint                 not null default 0,
     treasury_account_id    bigint                 not null,
+    type                   character varying(20) not null default 'NON_FUNGIBLE_UNIQUE',
```

Comment:
Same thing above, this should be `FUNGIBLE_COMMON` right?
```suggestion
    type                   character varying(20) not null default 'FUNGIBLE_COMMON',
```

Reply:
Yep, fixed.